Library/Homebrew/linkage_checker.rb

```diff
@@ -104,6 +124,7 @@ def display_normal_output
     display_items "Indirect dependencies with linkage", @indirect_deps
     display_items "Variable-referenced libraries", @variable_dylibs
     display_items "Missing libraries", @broken_dylibs
+    display_items "Missing dependencies", @broken_deps
```

Comment:
What about `missing installed dependencies` or `broken dependencies` or similar; it's not clear from this whether they are missing because they aren't installed or missing because they aren't specified in the formulae?